src/overview/actions.js

```diff
@@ -26,11 +26,17 @@ export const showDeleteConfirm = createAction(
 )
 export const hideDeleteConfirm = createAction('overview/hideDeleteConfirm')
 export const resetDeleteConfirm = createAction('overview/resetDeleteConfirm')
+export const resetSharePopup = createAction('overview/resetSharePopup')
 export const changeHasBookmark = createAction('overview/changeHasBookmark')
 export const incSearchCount = createAction('overview/incSearchCount')
 export const initSearchCount = createAction('overview/initSearchCount')
 export const setResultDeleting = createAction('overview/setResultDeleting')
-
+export const resetActiveShareIndex = createAction(
+    'overview/resetActiveShareIndex',
+)
+export const setActiveShareIndex = createAction('overview/setActiveShareIndex')
+export const truecopy = createAction('overview/truecopy')
```

Comment:
You don't need to make two different actions and reducer for this, you can set just be sending the bool.

Reply:
Fixed